src/ansi_formats.go

```diff
@@ -69,6 +70,7 @@ func (a *ansiFormats) init(shell string) {
 		a.escapeLeft = ""
 		a.escapeRight = ""
 	}
+	a.hyperlink = "\x1b]8;;%s\x1b\\%s\x1b]8;;\x1b\\"
```

Comment:
this needs to be escaped differently for zsh and bash, otherwise the line of the prompt won't be correct anymore and rendering is ruined. You can validate this by doing a tab completion using `cd` for example and notice the prompt just a bit off after pressing `TAB` the first time.